examples/lighting-app/mbed/mbed_app.json

```diff
@@ -30,7 +30,7 @@
                 "CORDIO"
             ],
             "led-active-state": 1,
-            "lighting-button": "USER_BUTTON",
+            "lighting-button": "BUTTON1",
```

Comment:
How this change is related with switching mbed to newer version?

Reply:
They change it in Mbed